lighthouse-cli/test/smokehouse/test-definitions/core-tests.js

```diff
@@ -70,8 +70,7 @@ const smokeTests = [{
   id: 'mixed-content',
   expectations: require('./mixed-content/mixed-content-expectations.js'),
   config: require('../../../../lighthouse-core/config/mixed-content-config.js'),
-},
-{
+}, {
```

Comment:
I thought we had some nice cors smoketests here did they disappear?

those are the ones I think are super valuable for us to validate the new approach will work too :)

Reply:
oh man i deleted it b/c the legacy js smoke covered fetching maps, but i totally forgot the point of it was the cors bit :P i'll resurrect it